src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -153,6 +159,7 @@ function OptionRowLHN(props) {
     const formattedDate = DateUtils.getStatusUntilDate(statusClearAfterDate);
     const statusContent = formattedDate ? `${statusText} (${formattedDate})` : statusText;
     const isStatusVisible = Permissions.canUseCustomStatus(props.betas) && !!emojiCode && ReportUtils.isOneOnOneChat(optionItem);
+    const isDraft = props.draftReportIDs[props.reportID];
```

Comment:
Could we rename this to hasDraft? We are going to bring a type of a report into the app which will be called Draft so isDraft will mean something else
```suggestion
    const hasDraft = props.draftReportIDs[props.reportID];
```

Reply:
Yes sure thing !